### ISSUE TITLE
NTR: HIV p24 antigen positive in blood (Lab: serology & immunological tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8763]
New term request HIV p24 antigen positive in blood (nadea3_181108174331):

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000798
- label: Positive bloodstream HIV p24 antigen test
- parents: [Positive bloodstream infectious agent antigen test (HP:0430109)](https://hpo.jax.org/app/browse/term/HP:0430109)
- pmid: PMID:7561186
- def: Demonstration of the p24 antigen of human immunodeficiency virus in the bloodstream.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8763
- synonyms:
